National Library of Norway.js

```diff
@@ -178,27 +297,99 @@ var testCases = [
 				"ISBN": "9788291375052",
 				"libraryCatalog": "National Library of Norway",
 				"numPages": "176",
-				"place": "Lesja",
 				"publisher": "Bondekvinnelaget",
 				"shortTitle": "Mat frå gard og grend",
-				"url": "http://urn.nb.no/URN:NBN:no-nb_digibok_2008030304011",
+				"url": "https://urn.nb.no/URN:NBN:no-nb_digibok_2008030304011",
 				"attachments": [],
 				"tags": [
 					"Foods",
 					"Gudbrandsdalen",
 					"History",
-					"Kokebøker",
 					"Lesja",
 					"Mattradisjoner",
 					"Merkedager",
 					"Preservation",
-					"Religiøse fester"
+					"Religiøse fester",
+					"baking",
+					"barsel",
+					"begravelser",
+					"blodmat",
+					"bryllup",
+					"brød",
+					"desserter",
+					"fisk",
+					"flatbrød",
+					"graut",
+					"gravøl",
+					"grøt",
+					"gudbrandsdalen",
+					"høytider",
+					"innmat",
+					"julebakst",
+					"julemat",
+					"kinning",
+					"kokebøker",
+					"korn",
+					"lefser",
+					"lesja",
+					"matlaging",
+					"mattradisjoner",
+					"melkemat",
+					"norge",
+					"oppskrifter",
+					"sending",
+					"setermat",
+					"slakting",
+					"stomp",
+					"takkebakst",
+					"tradisjoner",
+					"tradisjonsmat",
+					"tresking",
+					"vilt",
+					"ysting"
 				],
 				"notes": [
```

Comment:
could we concat notes? Multiple notes, each with one sentence, aren't very useful.

Reply:
Ok